.github/workflows/bounty-payout.yml

```
@@ -0,0 +1,110 @@
+
+jobs:
+  process-bounty:
+    permissions:
+        contents: read
+        issues: read
+    if: github.event.pull_request.merged == true
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check for linked issues with bounties
+        id: check-issues
+        uses: actions/github-script@v6
```

actionlint (1.7.7)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)